endoreg_db/models/media/video/video_file_ai.py

```
@@ -297,8 +297,43 @@ def _predict_video_pipeline(
         logger.info("Inference completed for video %s.", video.uuid)
     except Exception as e:
         logger.error("Inference failed for video %s: %s", video.uuid, e, exc_info=True)
-        # Raise exception
-        raise RuntimeError("Inference failed") from e
+        # CUDA-OOM Fallback: Speicher freigeben und CPU versuchen
+        try:
+            import torch, gc
+            is_oom = isinstance(e, (getattr(torch.cuda, 'OutOfMemoryError', RuntimeError), RuntimeError)) and (
+                'out of memory' in str(e).lower() or 'cuda out of memory' in str(e).lower()
+            )
+        except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)